android-agent/src/main/java/io/opentelemetry/android/SessionIdTimeoutHandler.java

```diff
@@ -25,7 +25,7 @@
  */
 final class SessionIdTimeoutHandler implements ApplicationStateListener {
 
-    private static final long SESSION_TIMEOUT_NANOS = TimeUnit.MINUTES.toNanos(15);
+    private long SESSION_TIMEOUT_NANOS = TimeUnit.MINUTES.toNanos(15);
```

Comment:
Fields should be camel case, and we want to favor immutability.

```suggestion
    private final long sessionTimeoutNanos;
```

Reply:
Done fixed ✅